kdump-lib.sh

```diff
@@ -995,6 +1002,9 @@ kdump_get_arch_recommend_crashkernel()
 	local _arch _ck_cmdline _dump_mode
 	local _delta=0
 
+	# osbuild deploys rpm on chroot environment. kdump-utils has no opportunity
+	# to deduce the exact memory cost on the real target.
+	maximize_crashkernel=$(is_ostree)
```

Comment:
osbuild is different from ostree. Osbuild can be used to build  normal guest image or ostree image.

Reply:
I think I need is_osbuild() here. And I will figure out a way to implement it. Thanks